framework/src/ble_manager/ble_manager_state.c

```diff
@@ -408,6 +403,26 @@ ble_result_e blemgr_handle_request(blemgr_msg_s *msg)
 
 	} break;
 
+	case BLE_CMD_WHITELIST_LIST: {
+		BLE_STATE_CHECK;
+
+		int i;
+		uint16_t count = 0;
+		blemgr_msg_params *param = (blemgr_msg_params *)msg->param;
+		ble_addr *addr = (ble_addr *)param->param[0];
+		uint16_t size = *(uint16_t *)param->param[1];
+		for (i = 0; i < SCAN_WHITELIST_SIZE; i++) {
+			if (g_scan_whitelist[i].use == SCAN_WHITELIST_IN_USE) {
+				memcpy(&addr[count++], &g_scan_whitelist[i].addr, sizeof(ble_addr));
+			}
+			if (count >= size) {
```

Comment:
do we need to check ">" also here? I guess size minimum will be > 0 ?

Reply:
My purpose was copying addr list to the array as it allowed.
I will change this.